src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  function handleTabID(tabId) {
+    return tabs.some(el => el.id === tabId) ? tabId : tabs[0].id;
+  }
```

Comment:
Why this function? It will always return `tabId`  as all possible values for `tabId` are in `tabs`